app/helpers/dag_json_helper.rb

```diff
@@ -0,0 +1,115 @@
+module DagJsonHelper
+  DAG_NAME_HOST_FILTER = "host_filter".freeze
```

Comment:
NIT: If you add this magic comment as very first line of this file:
```
# frozen_string_literal: true
```
You can actually remove all these `.freeze` method calls.

That comment basically says: "-every literal string declared in this file is 'frozen' by default"